2009-05-21  Jakub Jelinek  <jakub@redhat.com>

	* sysdeps/unix/sysv/linux/accept4.c (__NR_accept4): Don't define.

---
 sysdeps/unix/sysv/linux/accept4.c |    4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

--- a/sysdeps/unix/sysv/linux/accept4.c
+++ b/sysdeps/unix/sysv/linux/accept4.c
@@ -1,4 +1,4 @@
-/* Copyright (C) 2008 Free Software Foundation, Inc.
+/* Copyright (C) 2008, 2009 Free Software Foundation, Inc.
    This file is part of the GNU C Library.
    Contributed by Ulrich Drepper <drepper@redhat.com>, 2008.
 
@@ -24,8 +24,6 @@
 #include <sysdep-cancel.h>
 #include <sys/syscall.h>
 
-#define __NR_accept4                            288
-
 
 #ifdef __NR_accept4
 int
